pkgs/top-level/all-packages.nix

```diff
@@ -17108,6 +17108,8 @@ in
     gtk = res.gtk2;
   };
 
+  theme-obsidian2 = callPackage ../data/themes/obsidian2 { };
```

Comment:
Any reason for the move?

Reply:
Yes. There is an understanding that themes are best in the `data` category than in the `misc` one. New themes have been added there, and old themes are being moved when they are updated.